### PR TITLE
Add cypress test for announcement

### PR DESCRIPTION
**Pull request by tuilagio**
For `212 Frontend testing`

- Add test for announcement: add, edit, delete.
- ~There are some `cy.wait()` lines that are commented out to please the eslint lint-staged. Hopefully  they are not necessary for other machine.~

### CODE REVIEW

**Comment by dominic-heil** [comment]
`cy.wait` should never be neccessary on any machine. To "fix" these bugs, add loading states into the components where useEffects are used that fetch needed data.

**Comment by tuilagio** [comment]
> `cy.wait` should never be neccessary on any machine. To "fix" these bugs, add loading states into the components where useEffects are used that fetch needed data.

yeh just got rid of these

File: frontend/cypress/system/UseCases.spec.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ client
 	.setKey(Cypress.env("APP_KEY"))
 ;
 
+const APP_ANNOUNCEMENTS_COLLECTION_ID = Cypress.env("CYPRESS_APP_ANNOUNCEMENTS_COLLECTION_ID");
```
Comment (dominic-heil) [inline frontend/cypress/system/UseCases.spec.js:14]: This should be the following as cypress automatically adds the `CYPRESS_` part.
```
const APP_ANNOUNCEMENTS_COLLECTION_ID = Cypress.env("APP_ANNOUNCEMENTS_COLLECTION_ID");
```

**Review by dominic-heil — CHANGES_REQUESTED** [review]
Runs on my machine. One tiny change request in code, and please also add something like
```
    [...]
    "APP_ANNOUNCEMENTS_COLLECTION_ID": "61a7099351b82"
    [...]
```
in the cypress_system_tests.json. Once this was changes it LGTM!

**Review by dominic-heil — APPROVED** [review]
LGTM

**Review by steinkohl — APPROVED** [review]
LGTM

**Pull request merged**